src/app/containers/MediaPage/index.jsx

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable */
```

Comment:
Out of curiosity what is eslint complaining about in this file?

Reply:
No proptype validation :(

Reply:
Can you add some? :P 🙈

Reply:
Added some - this component is just a temporary stub though - we just need something for the route to render

The actual media page container is being done under a different issue, and will replace all of this